fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroup.java

```diff
@@ -103,7 +103,12 @@ private WorkloadGroup(long id, String name, Map<String, String> properties, long
             properties.put(ENABLE_MEMORY_OVERCOMMIT, properties.get(ENABLE_MEMORY_OVERCOMMIT).toLowerCase());
         }
         if (properties.containsKey(CPU_HARD_LIMIT)) {
-            this.cpuHardLimit = Integer.parseInt(properties.get(CPU_HARD_LIMIT));
+            String cpuHardLimitStr = properties.get(CPU_HARD_LIMIT);
```

Comment:
if user write code like cpu_hard_limit="1%    ", then the code is wrong. I think you should trim